### DISCUSSION TITLE
Is it possible to override deployment security settings for certain trusted users

### DISCUSSION THREAD

**Discussion by aniket0s** [goharbor/harbor#18532] [Q&A]
We are running Harbor 2.5 and are trying to integrate it with a third party security scanner. Currently we only allow users to pull images that are signed via Cosign, but this prevents the security scanner from being able to scan all the images in the project.

I saw a similar issue that seems to have been closed back in 2.2 with the introduction of system level robot accounts (https://github.com/goharbor/harbor/issues/11574). I tried using a system level robot account with all permissions granted, and it was still unable to pull unsigned images due to the deployment security settings.

Is there any way we can have the scanner account have access to pull all images, regardless of deployment security settings?

**Comment by stonezdj** [comment]
It is a requirement, you could create an issue for it.

**Reply by Vad1mo** [reply]
@aniket0s and place a reference here.
